enterprise-modules/charts/src/charts/chartComp/chartProxies/chartProxy.ts

```diff
@@ -198,14 +203,15 @@ export abstract class ChartProxy {
 
     public downloadChart(dimensions?: { width: number; height: number }, fileName?: string, fileFormat?: string) {
         const { chart } = this;
-        const imageFileName = fileName || (chart.title ? chart.title.text : 'chart');
+        const rawChart = deproxy(chart);
+        const imageFileName = fileName || (rawChart.title ? rawChart.title.text : 'chart');
         const { width, height } = dimensions || {};
 
         AgChart.download(chart, { width, height, fileName: imageFileName, fileFormat });
     }
 
     public getChartImageDataURL(type?: string) {
-        return this.chart.scene.getDataURL(type);
+        return deproxy(this.chart).scene.getDataURL(type);
```

Comment:
nitpick: I guess you could reuse `getChart()` here:

```suggestion
        return this.getChart().scene.getDataURL(type);
```